### ISSUE TITLE
citation update

### ISSUE THREAD

**Issue by vladdez** [behinger/eegvis#1]
The repo lacks cite me button and version.

Also consider that the name is not unique anymore https://github.com/eegml/eegvis

**Comment by behinger** [comment]
yeah, I guess it is time to archive this project anyway; I dont think I have capacity to support an old matlab toolbox :|

the name is unfortunate due to the other matlab eegvis toolbox also...
